lib/rubocop/cop/style/attr.rb

```diff
@@ -14,7 +14,24 @@ def on_send(node)
         end
 
         def autocorrect(node)
-          ->(corrector) { corrector.replace(node.loc.selector, 'attr_reader') }
+          _receiver, _method_name, attr_name, setter = *node
+          node_expr = node.loc.expression
+          attr_expr = attr_name.loc.expression
+          replacement = 'attr_reader'
```

Comment:
Changing this "default value" afterwards seems a bit confusing to me. I'd rather drop the default.